admin/build_targets/ubuntu-14.04/Dockerfile

```diff
@@ -4,15 +4,10 @@
 # environment.
 #
 
-FROM ubuntu:14.04
+FROM clusterhqci/fpm-ubuntu-trusty
 MAINTAINER ClusterHQ <contact@clusterhq.com>
-RUN ["apt-get", "update"]
-RUN ["apt-get", "install", "--no-install-recommends", "-y", "git", "ruby-dev", "libffi-dev", "libssl-dev", "build-essential", "python-pip", "python2.7-dev", "lintian"]
-# Despite being a packaging tool, fpm isn't yet packaged for Ubuntu
-# See https://github.com/jordansissel/fpm/issues/611 and
-# https://github.com/jordansissel/fpm/issues/657
-RUN ["gem", "install", "fpm"]
 COPY requirements.txt /tmp/
+RUN echo UMASK 022 >> /etc/logins.defs
```

Comment:
I think this can be removed. When creating Debian packages, lintian expects certain permissions (umask 022). We added this to the Dockerfile building these packages when it was actually needed in the script for provisioning an Ubuntu box and it has since been added there.